packages/itwin/tree-widget/README.md

```diff
@@ -71,6 +71,7 @@ While we expect this package to be mostly used with [AppUI](https://github.com/i
   - `createVisibilityTreeRenderer` returns a tree renderer that renders nodes with "eye" checkboxes. Its building blocks:
     - `createVisibilityTreeNodeRenderer`
     - `VisibilityTreeNodeCheckbox`
+  - `useVisibilityTreeState` is used to create and manage tree state.
   - `useVisibilityTreeFiltering` is used to filter the hierarchy.
```

Comment:
I think we should omit `useVisibilityTreeFiltering` from readme, since it's now deprecated